updateRepo.sh

```diff
@@ -1,34 +1,53 @@
+#!/bin/bash
+
 echo "Checking if repo needs to be updated"
 export ORG="IBM-Swift"
 export REPO="generator-swiftserver-projects"
 export GH_REPO="github.com/${ORG}/${REPO}.git"
-export BRANCH="init"
+export BRANCHES="init openAPI"
 export projectName="Generator-Swiftserver-Projects"
 
-cd ${TRAVIS_BUILD_DIR}
-mkdir current
-cd current
-git clone -b ${BRANCH} "https://${GITHUB_USERNAME}:${GITHUB_PASSWORD}@github.com/${ORG}/${REPO}.git"
-export currentProject=`pwd`
+for BRANCH in $BRANCHES
+do
+  cd "${TRAVIS_BUILD_DIR}" || exit
+  mkdir current
+  cd current || exit
+  git clone -b "${BRANCH}" "https://${GITHUB_USERNAME}:${GITHUB_PASSWORD}@github.com/${ORG}/${REPO}.git"
+  currentProject=$(pwd)
+
+  mkdir -p "${TRAVIS_BUILD_DIR}/new/${projectName}"
+  cd "${TRAVIS_BUILD_DIR}/new/${projectName}" || exit
+  if [[ "${BRANCH}" == "init" ]]
+  then
+    yo swiftserver --init --skip-build
+  elif [[ "$BRANCH" == "openAPI" ]]
+  then
+    yo swiftserver --app --spec '{ "appName":"'"$projectName"'", "appType":"scaffold", "appDir":".", "openapi":true, "docker":true, "metrics":true, "healthcheck":true }' --skip-build
+  fi
+
+  rm spec.json
+  echo "Generate README rtf"
+  pandoc README.md -f markdown_github -t rtf -so README.rtf
+  cd "${TRAVIS_BUILD_DIR}"/new || exit
+  newProject=$(pwd)
 
-mkdir -p ../new/${projectName}
-cd ../new/${projectName}
-yo swiftserver --init --skip-build
-echo "Generate README rtf"
-pandoc README.md -f markdown_github -t rtf -so README.rtf
-cd ../
-export newProject=`pwd`
+  currentRepo="${currentProject}/${REPO}"
+  newRepo="${newProject}/${projectName}"
 
-if diff -x '.git' -r ${currentProject}/${REPO} ${newProject}/${projectName}
-then
-  echo "Project does not need to be updated"
-  rm -rf ${currentProject}/${REPO} ${newProject}/${projectName}
-  exit 1
-fi
+  if diff -x '.git' -r "${currentRepo}" "${newRepo}"
+  then
+    echo "Project does not need to be updated"
+    rm -rf "${currentRepo}" "${newRepo}"
+    exit 1
+  fi
 
-echo "Project needs to be updated"
-cp -r ${newProject}/${projectName}/. ${currentProject}/${REPO}
-cd ${currentProject}/${REPO}
-git add .
-git commit -m "CRON JOB: Updating generated project"
-git push origin ${BRANCH}
+  echo "Project needs to be updated"
+  cp -r "${newProject}/${projectName}/." "${currentProject}/${REPO}"
+  cd "${currentProject}/${REPO}" || exit
+  git add .
```

Comment:
Would this handle the case where a file is deleted from the project?

Reply:
I don't think so, nor would it handle a new file I think. 
Again I wasn't why this was done that way before but I'll correct this to account for those cases.